west.yml

```diff
@@ -234,7 +234,7 @@ manifest:
       groups:
         - hal
     - name: hal_stm32
-      revision: 6443e5c4d391c3e9e283c5728cd07d855e663965
+      revision: pull/217/head
```

Comment:
Is this change for debug only? Is it discarded automatically?

Reply:
This one will be changed when the relative module PR on stm32 module will be approved/merged.